src/storage/src/statistics.rs

```diff
@@ -546,31 +647,298 @@ impl SinkStatistics {
         }
     }
 
+    /// Clear the data, resetting gauges to uninitialized.
+    /// This does not reset prometheus gauges, which will be overwritten with new values.
+    pub fn clear(&self) {
+        let mut cur = self.stats.borrow_mut();
+        cur.0.clear()
+    }
+
+    /// Get a snapshot of the data, returning `None` if all gauges are not initialized.
+    pub fn snapshot(&self) -> Option<SinkStatisticsUpdate> {
+        let cur = self.stats.borrow_mut();
+
+        match &cur.0 {
```

Comment:
This match is not matching anything

Reply:
Its to mirror the source case, which has gauges